src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java

```diff
@@ -66,17 +68,17 @@ public FileUploadSubmissionResource(SubmissionRepository submissionRepository, R
     }
 
     /**
-     * POST /exercises/{exerciseId}/file-upload-submissions : Create a new fileUploadSubmission
+     * POST exercises/{exerciseId}/file-upload-submissions : Create a new fileUploadSubmission
```

Comment:
```suggestion
     * POST exercises/:exerciseId/file-upload-submissions : Create a new fileUploadSubmission
```

Reply:
Even if this does not directly add a lot to the readability, I think we should work on making this uniform :)